test/functional/ex_cmds/source_spec.lua

```diff
@@ -60,6 +60,15 @@ describe(':source', function()
     eq('Vim(let):E15: Invalid expression: #{', exc_exec("'<,'>source"))
   end)
 
+  it('does not break if current buffer is modified while sourced', function()
+    insert [[
+      bw!
```

Comment:
prefer fully-qualified names in tests and scripts
```suggestion
      bwipeout!
```

Reply:
My bad, I'm so used to using `bw` that I sometimes forget it's shortened 😅